### ISSUE TITLE
Unable to publish

### ISSUE THREAD

**Issue by kuu** [vinsonchuong/dist-es6#5]
Hi, I've just started using dist-es6 for my project ([kontainer-js](https://github.com/kuu/Kontainer)).
However, the following error occurred on `npm publish`
- Steps:

```
$ npm version patch
$ npm publish
```
- Error:

```
+ kontainer-js@1.6.5
npm ERR! publish Failed PUT 403
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/n/versions/node/4.4.2/bin/node" "/usr/local/bin/npm" "publish"
npm ERR! node v4.4.2
npm ERR! npm  v3.8.5
npm ERR! code E403
npm ERR! "You cannot publish over the previously published version 1.6.5." : kontainer-js
```
- package.json:

```
  "version": "1.6.5",
...
  "scripts": {
    "prepublish": "npm run build && dist-es6"
  },
...
  "dependencies": {
    "dist-es6": "^0.1.13"
  }
```

I'm sure that 1.6.5 has never been published before.
Can you advise me on what could cause this?

Thanks

**Comment by kuu** [comment]
Sorry, it seems the package itself was correctly published. (I can install the published version from npm.)
So, the problem is only the error message.

**Comment by vinsonchuong** [comment]
Thanks for using `dist-es6`! There aren't many users using this yet. I'm sure that there are use cases that don't quite work 100%. So, I really appreciate the feedback.

That's a really weird error especially if you've never published that version before. Can you verify that the published code of `kontainer-js` is working correctly? Also, I noticed that you have a version `1.6.6` now. Did you see that error message again?

I'll try to publish some dummy projects to see if I can reproduce this.

Thanks for the help!

**Comment by kuu** [comment]
@vinsonchuong Thanks for the reply.
Yeah, the package itself is working correctly. I mean the code transpiled via dist-es6 was successfully published to npm as 1.6.6 and users can install the version.
But the error ("You cannot publish over the previously published version 1.6.6.") still occurred.

BTW, `dist-es6` is what I was looking for and I'm glad to find this today.
Thanks for the great tool!

**Comment by vinsonchuong** [comment]
@kuu, I was able to reproduce this issue, but then after fixing a different issue (vinsonchuong/dist-es6#6), I am no longer able to reproduce this one. Can you let me know if you're still seeing this with version `0.1.14`?

Thanks!

**Comment by kuu** [comment]
Hmm, I just published my project but the same error occurred:

```
+ kontainer-js@1.6.15
npm ERR! publish Failed PUT 403
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/n/versions/node/4.4.2/bin/node" "/usr/local/bin/npm" "publish"
npm ERR! node v4.4.2
npm ERR! npm  v3.8.5
npm ERR! code E403

npm ERR! "You cannot publish over the previously published version 1.6.15." : kontainer-js
```

The package itself seems to be successfully published.
So, just the error message.
Is my [package.json](https://github.com/kuu/Kontainer/blob/master/package.json) wrong?

**Comment by vinsonchuong** [comment]
Can you try removing the dist directory if it exists?

**Comment by kuu** [comment]
No I don't have dist directory in my project thanks to dist-es6.

**Comment by vinsonchuong** [comment]
Ok, I believe I fixed the problem in version 0.1.15. Let me know it you're still seeing the error.

**Comment by kuu** [comment]
Thanks for the update.
I tried `npm publish` with dist-es6 0.1.15.
But this time a different error was observed.

```
+ kontainer-js@1.6.25
Error: Command failed: /bin/sh -c pgrep npm

    at ChildProcess.exithandler (child_process.js:213:12)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:827:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)

npm ERR! addLocal Could not install /Users/kmiyazaki/GitHub/kuu/Kontainer
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/n/versions/node/4.4.2/bin/node" "/usr/local/bin/npm" "publish"
npm ERR! node v4.4.2
npm ERR! npm  v3.8.5
npm ERR! code ELIFECYCLE
npm ERR! kontainer-js@1.6.25 prepublish: `npm run build && dist-es6`
npm ERR! Exit status 1
```

The publishing itself went well. So, again, it's just an error message.
